spec/lib/vmdb/settings_spec.rb

```diff
@@ -293,54 +293,32 @@
 
           expect(SettingsChange.count).to eq 7
           expect(miq_server.settings_changes.find_by(:key => "/api/token_ttl")).to be nil
-          expect(Vmdb::Settings.for_resource(miq_server).api.new_key).to eq "new value"
-          expect(Vmdb::Settings.for_resource(second_server).api.token_ttl).to eq server_value
+          expect(described_class.for_resource(miq_server).api.new_key).to eq "new value"
```

Comment:
I know that these specs haven't really changed, but why are we even testing this condition here?

Reply:
@carbonin shouldn't we test that only specific key was deleted and other keys were not touched ?